workshops/forms.py

```diff
@@ -878,3 +878,51 @@ class Meta:
         fields = (
             'status', 'sent_date', 'paid_date', 'notes'
         )
+
+
+class TrainingRequestForm(forms.ModelForm):
+    intro = HTML("""
+        <p>Any text at the beginning of the form?</p>
```

Comment:
We run two-day instructor training classes for people who wish to learn how to teach Software Carpentry and Data Carpentry workshops. If you wish to apply to attend an upcoming workshop, please fill in the form below. We will notify applicants as spaces become available.  If you have any questions, please mail training@software-carpentry.org.

Please note that as a condition of taking this training:

1. You are required to abide by our code of conduct, which can be found at http://software-carpentry.org/conduct/.

2. You must complete three short tasks after the course in order to complete certification. The tasks are described at http://swcarpentry.github.io/instructor-training/checkout/, and take a total of approximately 2 hours.

3. You are expected to teach at a Software Carpentry or Data Carpentry workshop within 12 months of the course.

For more information on Software and Data Carpentry instructor training, please see http://swcarpentry.github.io/instructor-training/.